packages/providers/upload-aws-s3/README.md

```diff
@@ -30,6 +30,7 @@ npm install @strapi/provider-upload-aws-s3 --save
   - `signedUrlExpires` is the number of seconds before a signed URL expires. (See [how signed URLs work](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/private-content-signed-urls.html)). Defaults to 15 minutes and URLs are only signed when ACL is set to `private`.
   - `Bucket` is the name of the bucket to upload to.
 - `actionOptions` is passed directly to the parameters to each method respectively. You can find the complete list of [upload/ uploadStream options](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html#upload-property) and [delete options](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html#deleteObject-property)
+- `mapFileDataToUrl` optionally pass this function to define custom logic for building file url. This is provided as a way to handle the many different s3 provider options.
```

Comment:
```suggestion
- `mapFileDataToUrl` optionally pass this function to define custom logic for building file url from the uploaded file data. This will be the url that will be stored in the database. This is provided as a way to handle the many different s3 provider options.
```